src/LinearMaps.jl

```diff
@@ -1,116 +1,76 @@
 __precompile__(true)
 module LinearMaps
 
-export LinearMap, AbstractLinearMap
+export LinearMap
 
-import Base: +, -, *, \, /, ==, transpose
-
-if VERSION >= v"0.7.0-DEV.1415"
-    const adjoint = Base.adjoint
-else
-    const adjoint = Base.ctranspose
-end
+using LinearAlgebra
+import SparseArrays
 
+import Base: +, -, *, \, /, ==
```

Comment:
You can probably get rid of this line as well now, since you fully qualified all identifiers including operators.

Reply:
Just removing the line gives an error, and Julia is suggesting `You may have intended to import Base.*`. I'll try to `using Base` then...? The error occurs when I want to `A * v` in `sparse`.

Reply:
Maybe there is still some method definition for `*` somewhere that we've overlooked.

Reply:
I am fine merging like this. This looks great, and is by far the most polished PR I've ever seen. I could still look into this particular point once I have it here locally, but it is not important.

Reply:
The issue is that it errors when we use the symbols on the rhs, not when we are defining them. I think the function definitions are fine. It complains about line 62:
```
MethodError: no method matching *(::Array{Complex{Float64},2}, ::Array{Complex{Float64},1})
```
which sounds very weird. I can get rid of the error by replacing `A * v` by `Lv = Base.:(*)(A, v)`, but then it complains about the use of `\`. I guess one would have to fully qualify all operators throughout. Strangely, just `using Base` does not solve the issue, in contrast to `using LinearAlgebra`, where we are doing the same things: both using and extending functions.

Reply:
You're welcome. Was my pleasure, great package!